src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -148,12 +176,24 @@ export default class EnhancedReactTable extends React.Component<IEnhancedReactTa
                     handleInput={this.handleFilterInput}
                     onColumnToggled={this.handleVisibilityToggle}
                     showSearch={true}
+                    className="pull-right"
+                    paginationProps={{itemsPerPage:this.state.itemsPerPage,
```

Comment:
No need to rename the component itself :) We can have multiple instances of it if we think header is getting too crowded. We don't need to worry about it right now I guess.